### ISSUE TITLE
Skip empty ECS clusters instead of failing entire backend

### ISSUE THREAD

**Issue by zyzop** [containous/traefik#2168]
### Do you want to request a *feature* or report a *bug*?

bug

### What did you do?
I tried to use new ECS backend feature - autoDiscoverClusters.
So I added to my command line option `ecs.autodiscoverclusters=true`

It generally works, but with one remark.

I had one cluster without instances. 
Until I removed it, ecs backend wasnt working at all, giving me this error

```shell
"time="2017-09-22T19:01:33Z" level=error msg="Provider connection error InvalidParameterException: Container instance cannot be empty."
```

There wasn't even an ECS tab in traefik UI.
I believe this is a wrong behaviour, you should just skip empty clusters, not bring down whole backend.

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
1.4rc3
```

**Comment by jeffreykoetsier** [comment]
PR #2170

**Comment by traefiker** [comment]
Closed by #2170.
